package.json

```diff
@@ -2,14 +2,21 @@
   "name": "fortepiano",
   "version": "0.1.0",
   "description": "",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
-  "files": [
-    "lib/"
-  ],
+  "main": "./lib/index.js",
+  "module": "./esm/index.js",
+  "types": "./index.d.ts",
+  "sideEffects": false,
   "scripts": {
-    "build": "tsc",
-    "prepare": "npm run build",
+    "prebuild": "npm run clean",
+    "build": "npm run build:esm && npm run build:cjs",
+    "build:cjs": "tsc --build ./tsconfig.cjs.json",
+    "build:esm": "tsc --build ./tsconfig.esm.json",
+    "clean": "rm -rf ./dist/",
+    "eslint": "eslint --fix-dry-run ./",
+    "eslint:fix": "eslint --fix ./",
+    "prepublishOnly": "npm run build && ./scripts/hybridize.sh",
```

Comment:
When is meant to be executed this script? It is not present in GitHub workflows, so how and when is going to be executed?
And moreover, is there any more significant name? Maybe this is causing my misunderstanding.

Reply:
It's a standard one: https://docs.npmjs.com/cli/v8/using-npm/scripts#prepare-and-prepublish

Reply:
Ah, I didn't know that! Now it's pretty clear to me.